frameworks/g-cloud-9/questions/services/multiqDataSanitisationHosting.yml

```diff
@@ -0,0 +1,13 @@
+id: multiqDataSanitisation
```

Comment:
I *think* for cases like this you could have a single `multiqDataSanitisation` and list all four sub-questions in the `questions` list.  If the sub-questions depend on a particular lot then only the relevant sub-questions will be shown.

But maybe test I'm right about that.